tests/unit/amo/components/TestErrorList.js

```diff
@@ -58,36 +58,42 @@ describe(__filename, () => {
     );
   });
 
-  it('renders a reload button for signature expired errors', () => {
-    const _window = { location: { reload: sinon.stub() } };
-    const root = render({
-      _window,
-      code: API_ERROR_SIGNATURE_EXPIRED,
-      messages: ['Signature error'],
-    });
+  it.each(API_ERRORS_SESSION_EXPIRY)(
+    'renders a reload button for session expiry errors',
```

Comment:
To help with readability of the tests, you can include the code in the test description by including an `%s`, like this:
`'renders a reload button for session expiry error: %s'`.